randovania/games/samus_returns/logic_database/Area 3 - Factory Exterior.txt

```diff
@@ -31,7 +31,7 @@ Extra - asset_id: collision_camera_002
   * Extra - actor_name: Door002
   * Extra - actor_type: doorpowerpower
   > Chozo Seal
-      Grapple Beam or Climb Rooms Vertically (No High Jump)
+      Trivial
```

Comment:
What changed to make this trivial? lack of acid?

Reply:
Leftover from when acid was a thing. It's just a simple jump.